src/test/java/uk/gov/hmcts/reform/judicialapi/service/impl/JudicialRoleTypeServiceImplUnitTest.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.reform.judicialapi.service.impl;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.*;
+
```

Comment:
Remove * and add packages which are required

Reply:
This is done